_includes/tutorials/finding-distinct/.gitignore

```diff
@@ -0,0 +1,2 @@
+kstreams/code/tutorial-steps/dev/outputs/
```

Comment:
@bbejeck is this line intended to be part of this PR?

Reply:
Yes, see my comment below.  I found this by running a test for an issue the `kstreams/code/tutorial-steps/dev/outputs/` directory contains the output generated by the harness-runner and shouldn't get checked-in. cf https://github.com/confluentinc/kafka-tutorials/issues/729